packages/melody-runtime/src/filters.js

```diff
@@ -961,3 +961,79 @@ export function strtotime(input, now) {
 
     return date.getTime() / 1000;
 }
+
+export function trim(str, charlist, side = 'both') {
```

Comment:
Since this implementation is quite a bit slower than the native one (~87% slower) I'd suggest to check if `charlist` is provided and if its not, to return `str.trim()`.

Reply:
Yes good idea! I think the native trim will cover the majority of use-cases.